magento/magento2
# Add The Ability To Limit Autocomplete Results

### Description

Adds the ability to limit the number of results in the search autocomplete. Defaults to 8

![image](https://user-images.githubusercontent.com/4132322/31801557-1a0c6a64-b517-11e7-8b08-b72bc75b58f3.png)

### Manual testing scenarios

1. In the admin create a product called "Elephantawesomepants"
2. On the frontend search for "Ele", "Elep", "Eleph", etc... until you've passed the default limit
3. Notice that no more than the configured limit of search terms are displayed

### Contribution checklist
